DEV-2/StringCreator.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DEV_2
+{
+    public static class StringCreator
+    {
+        public static string CreateFromConsole()
+        {
+            Console.WriteLine("Input string");
+            string str = Console.ReadLine();
+            return str;
+        }
```

Comment:
Too much code copy paste in three methods, try put the similar part to a separate method